src/libm/sleefsimddp.c

```diff
@@ -289,7 +298,7 @@ EXPORT CONST vdouble xsin(vdouble d) {
 					vor_vo_vo_vo(visnegzero_vo_vd(r),
 						     vgt_vo_vd_vd(vabs_vd_vd(r), vcast_vd_d(TRIGRANGEMAX)))),
 		       vcast_vd_d(-0.0), u);
-  
+
```

Comment:
Please don't change this line, we don't have a coding standard that enforces to remove lines with spaces.